acct.bionimbus.org/manifest.json

```diff
@@ -63,15 +58,15 @@
   "guppy": {
     "indices": [
       {
-        "index": "acct_etl",
-        "type": "subject"
+        "index": "acct.bionimbus.org_file",
+        "type": "file"
       },
       {
-        "index": "acct_file",
-        "type": "file"
+        "index": "acct.bionimbus.org_subject",
+        "type": "subject"
       }
```

Comment:
?

Reply:
This is just matching the Elastic Search index name that was also set on etlMapping.yaml.
This is working by design.
Our automation names the Elastic Search index according to the web address / hostname of the target environment.
This should be harmless.
--
💡 
Things will work fine as long as the names of the indices between the guppy config in `manifest.json` and `etlMapping.yaml` match.